pkg/daemon/ceph/client/pool.go

```diff
@@ -357,6 +369,14 @@ func createStretchedReplicationCrushRule(context *clusterd.Context, clusterInfo
 		return errors.Wrap(err, "failed to get crush map")
 	}
 
+	// Check if the crush rule already exists
+	for _, rule := range crushMap.Rules {
```

Comment:
Small helper function with unit tests for the block?

Reply:
Missed?

Reply:
It's covered now by the `testCreateStretchCrushRule()` test, which tests both for crush rules that already exist and new rules.